randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1149,7 +1149,9 @@ Extra - room_id: [28]
   * Open Passage to Reservoir East/Other to Waterway
   * Extra - door_idx: (4,)
   > Pickup (Underwater Missile Tank)
-      Morph Ball
+      All of the following:
+          Morph Ball
+          Gravity Suit or After Sector 4 Water Level Lowered or Underwater Wall Jump (Expert)
```

Comment:
this should be organized into two things:
1) can move up
2) can move this without damage.
So i'd have 
```
ALL
- Morph
- ANY:
  - Gravity or After S4 lowered or UWJ
- ANY:
  - After s4 lowered or electric damage

Reply:
HJ should also work ^ tack that on to the above change